ui/src/components/delete/QasDelete.stories.js

```diff
@@ -0,0 +1,72 @@
+import QasDelete from './QasDelete.vue'
+
+export default {
+  component: QasDelete,
+  title: 'Components/Delete',
+
+  parameters: {
+    docs: {
+      description: {
+        component: 'Insert line breaks where newlines `\n` occur in the string.'
+      }
+    }
+  },
+
+  argTypes: {
+    dialog: {
+      description: 'recebe as configurações do dialog'
+    },
+    customId: {
+      description: 'CustomID receives a customized id of the item to be deleted or if not the id to be used will be from the route'
+    },
+    entity: {
+      description: 'The entity is used by vuex as a url to define which element will be deleted from the store. has as parameter the ID.'
+    },
+    label: {
+      description: 'Label is an attribute for the element to display text in its content'
+    },
+    tag: {
+      description: 'The tag defines what is the element type in the DOM'
```

Comment:
se estamos dentro do objeto `tag` não acha que fica redundante ficar começando `the tag...`?

Reply:
mudamos para outra definição